app.py

```diff
@@ -59,6 +60,14 @@ def initialize_database(autoinitialize=True):
     for logger in logging.getLogger().handlers:
         app.log.info("Logs are going to %r", logger)
 
+    # Register an error handler that logs exceptions through the
+    # normal logging process and tries to turn them into Problem
+    # Detail Documents.
+    h = ErrorHandler(app, app.config['DEBUG'])
+    @app.errorhandler(Exception)
+    def exception_handler(exception):
+        return h.handle(exception)
```

Comment:
Is it necessary to put this in `app.before_first_request`? It probably doesn't matter, but I don't think we do that in the circ manager.

Reply:
My thinking is that `before_first_request` is the first place where we know the correct value for `app.config['DEBUG']`. It's not that way in the circ manager but it probably should be.